src/systems/filecoin_files/data_transfer/_index.md

```diff
@@ -3,4 +3,92 @@ menuTitle: Data Transfer
 title: "Data Transfer in Filecoin"
 ---
 
+_Data Transfer_ is a system for transferring all or part of a `Piece` across the network when a deal is made.
+
+# Terminology
+
+- **Push Request**: A request to send data to the other party
+- **Pull Request**: A request to have the other party send data
+- **Requestor**: The party that initiates the data transfer request (whether Push or Pull)
+- **Responder**: The party that receives the data transfer request
+- **Data Transfer Voucher**: A wrapper around storage or retrieval data that can identify and validate the transfer request to the other party
```

Comment:
at first i wasn't in love with the word voucher, but i think it's pretty great:

> **voucher (n.)** a small printed piece of paper that entitles the holder to a discount, or that may be exchanged for goods or services.

i think this is much better than the prior name (auth token)

Reply:
(this could also just come later, post mainnet)